src/DataCollection.Shared/ViewModels/OriginRelationshipViewModel.cs

```diff
@@ -232,132 +79,11 @@ public ICommand SaveEditsCommand
                 return _saveEditsCommand ?? (_saveEditsCommand = new DelegateCommand(
                     async (x) =>
                     {
-                        var feature = await EditViewModel.SaveEdits(PopupManager, RelatedTable, null);
-
-                        if (feature != null)
-                        {
-                            // if a new record was added, we want to refresh the list so it shows up in the UI
-                            GetAddedRecordAndRefresh(feature);
-                        }
-
+                        var feature = await EditViewModel.SaveEdits(PopupManager, FeatureTable, null);
```

Comment:
It looks like this variable is never used, is that intentional?

Reply:
Changed it to save the value in the Feature property